config/opal_get_version.m4

```diff
@@ -99,3 +100,27 @@ m4_define([OPAL_GET_VERSION],[
                  [AC_MSG_RESULT([$$2_REPO_REV])])
     fi
 ])
+
+# OPAL_GET_MPI_STANDARD_VERSION(version_file)
+# -----------------------------------------------
+# parse version_file for MPI Standard version information, setting
+# the following shell variables:
+#
+#  MPI_VERSION
+#  MPI_SUBVERSION
+m4_define([OPAL_GET_MPI_STANDARD_VERSION],[
```

Comment:
sigh; we've broken this already, but there's no reason to use shell code at configure time.  There's already a build dependency of configure on VERSION, so we can just do this all during Autoconf time (similar to how we get the version to pass to AC_INIT).  No need to run a giant sed across an ever-growing version file every time someone runs configure...

Reply:
Yeah. I didn't want to tackle that in this change. Just trying to match what was already there.